### ISSUE TITLE
Create a Staging version of `https://pay.ubq.fi/secret`

### ISSUE THREAD

**Issue by Draeieg** [ubiquibot/QA-Improvements#2]
In order to fully test some features we need to generate bot config, the standard way is using our [Secret](https://pay.ubq.fi/secret) setup, however this only applies to production as a result we have no way of testing anything that should be interfaced using it unless we briefly merged into production 

By example in order to test [This](https://github.com/ubiquity/ubiquibot/pull/383#issuecomment-1608559842)

### What should be done 

 - Create a copy of [Secret](https://pay.ubq.fi/secret) that uses development instead of production 
 - we can call it https://pay.ubq.fi/secretqa or https://pay.ubq.fi/staging

**Comment by pavlovcik** [comment]
Does this organization have the permit wallet set up? Are these payment permits valid?

If so I think that we should consider a separate permit wallet. Also that would mean that you'll need approvals from me to fund bounties. You're free to set priority level but let me set time level or vice versa.

**Comment by Draeieg** [comment]
> Does this organization have the permit wallet set up? Are these payment permits valid?
> 
> If so I think that we should consider a separate permit wallet. Also that would mean that you'll need approvals from me to fund bounties. You're free to set priority level but let me set time level or vice versa.

Nope, those labels are just there for me to test if my assumptions are accurate

**Comment by Draeieg** [comment]
@wannacfuture this is your next task
